### DISCUSSION TITLE
[IWD]Painting with AI  Celebrate International Women’s Day Run AIGC in three steps using OpenVINO™

### DISCUSSION THREAD

**Discussion by openvino-dev-samples** [openvinotoolkit/openvino_notebooks#879] [Show and tell]
![MicrosoftTeams-image](https://user-images.githubusercontent.com/91237924/223342597-daf92aa2-16e1-4733-90d9-c23afcc358a6.png)
The OpenVINO™ toolkit team invites you to celebrate the 113th International Women's Day, by participating in “Painting with AI” event, where you will use innovative tools and technology to promote gender equality and embrace diversity in the AI community! 

OpenVINO™ helps reduce the AIGC usage threshold and accelerate model deployment. The event aims to pay tribute to the power of women in AI. 

### 1. **How to participate**
Join the fun now by creating a painting using OpenVINO™. Get Started in [three simple steps](https://github.com/openvinotoolkit/openvino_notebooks/issues/880).  
Choose one of the themes below and submit before the deadline. 

 **Painting themes:**
- [ ] Centered around the theme of "DigitALL: Innovation and technology for gender equality", conveying a positive message. 
- [ ] Pay tribute to International Women's Day by painting a bouquet of flowers and celebrate the holiday together! 

**Submission:** 
Post the screenshot of your painting and description in this discussion. [Example](https://github.com/openvinotoolkit/openvino_notebooks/discussions/879#discussioncomment-5186572)

### 2. **Technical requirements** 
You should use [stable diffusion notebook](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/225-stable-diffusion-text-to-image) to generate the picture.

### 3. **Schedule**
| Date | Milestones |
| :---- | :---- |
| March 8th | Opening  |
| March 8th - March 15th 23:59 (GMT+0) | Post painting and description  |
| March 15th 23:59 (GMT+0) | Deadline for submitting  |
| March 15th - March 20th 23:59 (GMT+0) | Public voting (thumbs-up your favorite painting)   |
| March 20th 23:59 (GMT+0) | Voting closes, the top 30 participants with the most  "👍" will receive surprise gifts  |

### 4. **Rules**

1) Participants need to create a painting related to **International Women's Day 2023 using OpenVINO™** and explain the meaning. 

2) The painting must be related to either of the **two themes** mentioned above. 

3) The submission deadline is **March 15th, 23:59 (GMT+0)**. 

4) The submitted results must include the **screen shot of painting with notebook interface, prompt and painting's description**. 

5) Based on the number of "👍", the top 30 paintings in this discussion will receive a **souvenir**.

**Comment by openvino-dev-samples** [comment]
### **Example**

**Prompt**: This place is side of beautiful river with nice trees around

**Screenshot**:
<img width="238" alt="image" src="https://user-images.githubusercontent.com/91237924/222651875-64b6d7ab-43ed-44f2-a86a-3f8d5128b43c.png">

**Parameter**: 
- seed: 42
- steps:50

**Description**: This is the place I wish to live at

**Comment by openvino-dev-samples** [comment]
**Prompt**: Depict a modern society with innovative technology, where both men and women are using these technologies to participate equally in various fields and activities. The painting can include various tech devices such as smartphones, tablets, VR headsets, etc. The scene can be colorful, showing the positive impact of new technology on life, as well as the concept of gender equality.

**Screenshot**:
<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/91237924/223592867-bfa40cd4-f0db-43e5-b1ba-d27c1d5dc9b3.png"> 
</p>
<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/91237924/223592896-0b59f6a2-9693-43c4-8798-2f56b894db0b.png"> 
</p>
<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/91237924/223592914-8e3b6ba0-f3cb-4e95-bf4f-c7cd5836ec6b.png"> 
</p>

**Comment by openvino-dev-samples** [comment]
**Prompt**: Describe a scene where a woman is innovating in the tech field. The painting can show a woman working in a laboratory, studio, or in front of a computer, surrounded by various tools and equipment. This can show the innovation and leadership of women in the tech field, and the power of technology to promote gender equality.
**Screenshot**:
<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/91237924/223593073-f161a1bb-577f-47ab-887a-4a913ef3dfbb.png"> 
</p>
<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/91237924/223593061-b9774da3-46af-47b5-a915-c3c68605fc14.png"> 
</p>
<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/91237924/223593226-7d339d5e-1271-42f3-9c67-9f94ff0281be.png"> 
</p>

**Comment by ceciliapeng2011** [comment]
**Prompts**: depict how women world cup would be in space within 100 years in future.

**Screenshot**:
![result (2)](https://user-images.githubusercontent.com/24786748/223625328-528163eb-88b3-4143-88fd-1fb251663128.png)
**Parameter**:

seed: 1000
steps:25

**Description**: I would like to see if AI could foresee the future~~ LOL.

**Comment by AnishaUdayakumar** [comment]
Prompt: Super Woman standing in cyberpunk cityscape like Tokyo New York  with tall buildings at dusk golden hour cinematic lighting

Screenshot:
![IWD Women](https://user-images.githubusercontent.com/109281183/223670195-b27e9f3b-c6fa-4eb9-95ee-72930755ae8f.png)
 
Parameter:
•	seed: 4370967
•	steps:30
Description: Envisioning my future techno city sunsets

**Comment by EvanYellow** [comment]
**Prompt:** a vibrant digital painting of a bouquet of flowers in celebration of International Women's Day, featuring bold and bright colors, intricate details, and a variety of flower types including roses, lilies, and daisies. The composition should be balanced with an emphasis on the center focal point while also showcasing each individual bloom.

**Screenshot:**
<img width="383" alt="image" src="https://user-images.githubusercontent.com/1029159/223730652-b8a6534e-9daf-4d74-9ffa-7dd98ea96244.png">

**Parameter:**
seed:42
steps:20

**Description:** Pay tribute to International Women's Day by painting a bouquet of flowers and celebrate the holiday together!

**Comment by JaneConan** [comment]
Prompt: Hats off to International women's Day, paint her a bouquet of flowers and celebrate the holiday topether! Dipital inclusion: Innovatiorand technology drive gender equality and mean positivity.
Screenshot:
![677f116e0d6a6cabd89a72506ec323e](https://user-images.githubusercontent.com/26090153/223767862-eafc5ff3-752f-4eb9-9b76-82d60c917d92.png)

![4653e6042ce91cd7252fa0ef262083e](https://user-images.githubusercontent.com/26090153/223767949-abc75d07-fb9e-4897-9f82-343d84be6f25.png)


Parameter:
seed:42
steps:20

Description: You are my sunshine!

**Comment by raymondlo84** [comment]
![image](https://user-images.githubusercontent.com/1720147/223868901-609d59d7-b2fa-4faa-992d-b929978fcf79.png)
Parameter:
seed: 42
steps: 40
Description:
![image](https://user-images.githubusercontent.com/1720147/223869230-e4257a50-e0d2-484c-bf28-273e0d64d0b6.png)

**Comment by paularamo** [comment]
**Prompt:** A hero women in tech - A garden with women in tech
**Screenshot** 
![image](https://user-images.githubusercontent.com/10940214/223869974-9dd37ab4-9e6e-4a2f-8596-5f3a1eb7fe83.png)
Parameter:
seed: 4000000
steps: 35

**Comment by gltzgl** [comment]
Prompt: Super Woman standing in sunset clouds, seashore,nightsky, sharp focus,matte, elegant，the most beautiful image ever seen,beautiful,attractive, friendly, casual, delightful, intricate, gorgeous, female, octane render, unreal engine, photograph, realistic skin texture, photorealistic, hyper realism,highly detailed

Screenshot:
![result_i2i](https://user-images.githubusercontent.com/45555181/223890835-fb0bae50-f3d4-4bcb-9514-09d4e8f37c33.png)


Parameter:
• seed: 42
• steps:10
Description: Independent and confident women

**Comment by v-leihe** [comment]
Prompt: Describe a scene where a woman is innovating in the tech field. the woman sat at her desk, her eyes fixed on a computer screen. she typed furiously on her keyboard
Screenshot:
![image](https://user-images.githubusercontent.com/13790811/225264397-f3c2f840-95dc-4d4f-84f6-2dbb52ad5d7f.png)
image
![result](https://user-images.githubusercontent.com/13790811/224538941-4a39bcc0-837d-4c4a-a177-14fabe86b837.png)

another try
![result3](https://user-images.githubusercontent.com/13790811/225265402-f46c576b-3d27-42f6-a2f3-fc297b5a5d42.png)
![result4](https://user-images.githubusercontent.com/13790811/225265513-502e41da-e725-4be4-aa93-4f9c97cc3f41.png)

Parameter:
seed:50000
steps:50

**Comment by natureLanguageQing** [comment]
Example
Prompt: 	A girly girl is someone who whole-heartedly embraces her femininity, without sacrificing her personality or strength
	 She cares about her behavior, style and appearance, but is never self-centered or mean

Screenshot:
![image](https://user-images.githubusercontent.com/37429336/224642552-4db1b322-b465-40ec-be2c-aeca24412b23.png)

Parameter:

seed: 42
steps:50
Description: A girly girl is someone who whole-heartedly embraces her femininity, without sacrificing her personality or strength
	 She cares about her behavior, style and appearance, but is never self-centered or mean

**Comment by fiorezhang** [comment]
Prompt: photo of a happy girl with navy-blue jacket, headset, in a cyberpunk computer laboratory, holding a rose flower, intricate, masterpiece
Seed: 8050307
Steps: 38

Final Image: 
![result](https://user-images.githubusercontent.com/25143601/224730360-967ee5e4-cc9a-4360-a408-3c39266fb660.jpg)

Description: 
Wish women in technology keep confidence and happy everyday. 

Screenshots: 
![Screenshot 2023-03-13 221516](https://user-images.githubusercontent.com/25143601/224730506-8850a4fb-e768-4c45-b69b-769d2942ec30.jpg)
![Screenshot 2023-03-13 221534](https://user-images.githubusercontent.com/25143601/224730536-5ba93562-18d3-4007-9962-1a3ac70f9b7f.jpg)
![Screenshot 2023-03-13 221701](https://user-images.githubusercontent.com/25143601/224730557-ab48351d-e548-4ff4-acfb-3a3d797c2c68.jpg)

**Comment by sleeplessai** [comment]
First time play Stable Diffusion model, even the first try of AIGC.

Runing screenshot:
![image](https://user-images.githubusercontent.com/2018063/225034391-8a45f08b-48f7-468e-a849-33c146575604.png)
![image](https://user-images.githubusercontent.com/2018063/225033612-808ef812-cc2c-4dea-acc1-d2c86341b055.png)

Final image:
![0f0b1f7786a90803600950e03716950](https://user-images.githubusercontent.com/2018063/225033494-89d53864-6bb8-489b-8235-cc93f4a92c38.png)

**Comment by RockL888** [comment]
![e484a821f76e178e46005073ae2583ae](https://user-images.githubusercontent.com/63142323/225185199-5b036450-95f5-4d37-88d1-95e0606639ef.jpg)

text_prompt:Regardless of your gender and where you come from, if you want to do something, please do it boldly.Celebrate women developers with openvino and flowers

seed: 42   steps:  20

implied meaning：At present, the prejudice of the social environment against female developers will unconsciously and negatively affect women's ability to exert themselves. We should encourage and affirm them.

**Comment by aadhamm** [comment]
[](url)Prompt: Celebrate International Women's Day with a bouquet of flowers that symbolizes the strength, resilience, and beauty of women. Deep green leaves, delicate pink and white carnations, bold red and yellow roses, and striking purple irises represent diversity, power, and hope. Let's honor women and their contributions to society, and work towards a world where every woman can thrive.

Screenshot:
![WhatsApp Image 2023-03-16 at 9 11 29 PM](https://user-images.githubusercontent.com/86083079/225729069-8fd4c02a-f8df-49d5-8d99-b34fd7db72f8.jpeg)

Parameters:
seed: 1678993775
steps: 50

**Comment by openvino-dev-samples** [comment]
We would like to offer you the opportunity to receive **an Intel Backpack** during **March 21st to April 10th, 2023.** 

If you wish to receive the Intel Backpack, please provide the information your full name, mobile/home phone number, your address, and Postal Code/Zip Code to Shanshan.wu@intel.com. 

Your personal data will be used for the purpose of ordering and delivering the Intel Backpack and Intel will only keep the information for so long as necessary to achieve that purpose. 

If you have any questions, please contact Shanshan Wu, email address: **Shanshan.wu@intel.com.** 

For more information on how Intel manages your personal data see [the Intel Privacy Notice](https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html). 

If you decide to order a gift for a family member, by providing the required information about your family members you agree to Intel using that information for these purposes.
